src/playlist-loader.js

```diff
@@ -203,6 +206,11 @@ export default class PlaylistLoader extends EventTarget {
     this.hls_ = hls;
     this.withCredentials = withCredentials;
 
+    const options = hls.options_;
+
+    this.parsers = (options && options.parsers) || [];
```

Comment:
Was there a request to expose the custom parser API as well?

Reply:
I think we might as well expose it, it's pretty useful